### ISSUE TITLE
Staging Review finding: Upload screens marked "Required" do not include required fields

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#93004]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IVC Forms
**Product:** 10-10d CHAMPVA Application Form
**Feature:** new product

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

Multiple pages in the applicant information section ask for documentation upload and indicate `(Required)` in the header, but there is no other indication that the field is required, and users are allowed to proceed through the form without uploading anything.

There are opportunities later in the form to add these uploaded files, but there is no indication on the individual upload pages that there will be another opportunity later in the form to upload a file. 

While viewing a page to upload a file, it could be confusing to see "required" in the header but not actually need to upload a file. Without the information that there is a chance later in the form to add the file, users might be confused and not sure how to proceed, or they might see this as a trust/reliability issue with VA.gov (that the page says a file is required, but then doesn't enforce it).

<details><summary>Link, screenshot or steps to recreate</summary>

![Applicant name and date of birth page, with two applicants entered and edit buttons alongside the names of each applicant.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/70913-Brian-1.png?raw=true)

</details>

## Recommended action

Typically, teams using patterns like this will use the name entered for the `aria-label`, eg. "Edit John Doe." If that's not feasible, you could also keep it generic but include some index, eg. "Edit applicant 1" --- not ideal, but each button would have a unique name and would be a bit easier to identify.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG21/#name-role-value)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by baltes** [comment]
Holding off while we think through duplicate upload guidance (e.g. uploading the same insurance card for more than one applicant) and other implications.

**Comment by jamiefiore** [comment]
This will be resolved in the update 1010d/OHI merge so this can be closed @andrea-merrill @RenataKeck @ManOnTheMooney
